src/main/java/com/sap/oss/phosphor/fosstars/owasp/data/Cvssv2.java

```diff
@@ -1,4 +1,4 @@
-package com.sap.oss.phosphor.fosstars.data.artifact.experimental.owasp.data;
+package com.sap.oss.phosphor.fosstars.owasp.data;
```

Comment:
Please rename the package to `com.sap.oss.phosphor.fosstars.data.artifact.owasp.data`. If you don't like the second `data` in the package name, you can use something like `domain` or `model`.